src/include/buffer.h

```diff
@@ -990,6 +990,7 @@ inline namespace v14_2_0 {
         _carriage = &always_empty_bptr;
         _buffers.clone_from(other._buffers);
         _len = other._len;
+        last_p = begin();
```

Comment:
There is a PR knocking `last_p` entirely to improve `bufferlist`'s cache density (the iterator is surprisingly large) and lower the cost on instance initialization. Let's resurrect it.